src/ucm/bistro/bistro.h

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2018.       ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+
+#ifndef UCM_BISTRO_BISTRO_H_
+#define UCM_BISTRO_BISTRO_H_
+
+#include <stdint.h>
+
+#include <ucs/type/status.h>
+
+typedef struct ucm_bistro_restore_point ucm_bistro_restore_point_t;
+typedef ucm_bistro_restore_point_t *ucm_bistro_restore_point_h;
+
+#if defined(__powerpc64__)
```

Comment:
maybe we should use bistro_x86_64.inl etc. ? 
really would like to avoid arch-specific #ifdef in more than one place..

Reply:
added separate headers